pegasus/sites.v3/code.org/public/css/logos.css

```diff
@@ -0,0 +1,20 @@
+/* 970px and down (tablet and phones): show 3 columns */
+
+@media screen and (min-width: 0px) and (max-width: 970px)
```

Comment:
Isn't the range for this technically 513px to 970x since 512px and below will be dealt with below?

Reply:
Yes. I'm following the comment style in `brstrap.css` here. What is a better way to communicate it, by updating comment or updating `min-width`?

Reply:
Oh this is referring to the actual range you specify in the `@media` query.  If you know you're going to be immediately overridden, you can avoid that by specifying correct `min-width` values.  That helps avoids seeing lots of CSS overriding other CSS when debugging.  Not a big deal, but something to keep in mind.

Reply:
Got it. I'll make it clearer.